portfolio/src/main/webapp/index.html

```diff
@@ -14,13 +14,30 @@ <h1>My Portfolio</h1>
          at MIT, where I just finished my first year.</p>
 
       <p>In high school, I had the chance to try several sports, two of which have played 
-         a pretty big part in who I am: swimming and rowing.</p>
+         a pretty big part in who I am: swimming and rowing. Click below for more information!</p>
 
       <div>
         <button onclick="toggleVisibility('swimming')">Swim</button>
         <button onclick="toggleVisibility('rowing')">Rowing</button>
-        <div class="invisible" id="swimming">My swimming life</div>
-        <div class="invisible" id="rowing">My rowing life</div>
+        <div class="invisible" id="swimming">
+          I began swimming at the age of 12. For most sports, I think that's pretty normal. For swimming, it's super
+          late. Accordingly, I was never too good at it. However, the sport did teach me most of what I consider to 
+          be my self-discipline, and is probably the reason I joined the crew team. Throughout high school, I found
+          a family in the swim team, making me grateful for sticking with it. 
+          <img src="images/swimming.JPG" width="500">
+        </div>
+        <div class="invisible" id="rowing">
+          I began rowing as a freshman in high school, where we could row on the (usually) beautiful Mercer Lake, 
+          home to the women's national team. Somehow falling in love with the long hours and quirky nature of the sport 
+          (e.g. going backwards), I continued on to race on the varsity squad in my junior and senior years. 
+          <br><img src="images/rowing.jpg" width="500"><br>
```

Comment:
nit: please add an alt attribute to images

https://google.github.io/styleguide/htmlcssguide.html#Multimedia_Fallback